persons.ps1

```diff
@@ -28,9 +37,9 @@ function get_system_metadata {
         Write-Information "Retrieving Access Token"
            
         $AccessToken = (get_oauth_access_token `
-                -BaseURI $config.BaseURI `
-                -ClientKey $config.ClientKey `
-                -ClientSecret $config.ClientSecret).access_token
+            -ClientKey $config.ClientKey `
+            -ClientSecret $config.ClientSecret `
+            -IntegrationKey $config.IntegrationKey).access_token
```

Comment:
This value is not difined in the configuration.json can you add this value (maybe with a example)?

$config.IntegrationKey